keymaps/terminal.json

```diff
@@ -2,7 +2,7 @@
   "atom-workspace": {
     "cmd-shift-t": "terminal:open"
   },
-  ".terminal-view": {
+  "terminal-view": {
```

Comment:
We are not registering a custom element, we are just using a custom element name which is completely permitted and actually makes the DOM elements panel nicer to look at. Plus Atom's using `atom-text-editor` and `atom-workspace` and others so this just makes it aligned with the core more